pkgs/by-name/no/notonoto/package.nix

```
@@ -14,10 +17,29 @@ stdenvNoCC.mkDerivation rec {
     hash = "sha256-1dbx4yC8gL41OEAE/LNDyoDb4xhAwV5h8oRmdlPULUo=";
   };
 
+  nativeBuildInputs = [
+    fontforge
+    python3Packages.setuptools
```

Don't need setuptools here this was a missing dependency in ttfautohint-py fixed in #378514.
Added the version of python where we added the decencies before instead. 
```suggestion
    python3'
```